src/pages/Home/Home.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {ThisDay} from "./components/ThisDay/ThisDay";
+import {ThisDayInfo} from "./components/ThisDayInfo/ThisDayInfo";
+import {Days} from "./components/Days/Days";
+import {HomeWrapper} from "./HomeStyle";
+
+interface Props {
+
+}
+ export const Home = (props: Props) => {
+  return (
+    <div>
```

Comment:
You can use <></> instead of <div>
```suggestion
    <>
```